config/config.go

```diff
@@ -845,6 +847,7 @@ func DefaultConsensusConfig() *ConsensusConfig {
 		CreateEmptyBlocksInterval:   0 * time.Second,
 		PeerGossipSleepDuration:     100 * time.Millisecond,
 		PeerQueryMaj23SleepDuration: 2000 * time.Millisecond,
+		DoubleSignCheckHeight:       int64(10),
```

Comment:
I think it may be unintuitive to enable this by default. For example, if I'm just playing around with a testnet then I won't be able to do a full chain restart since all nodes will panic. I'd vote for disabling this by default, and validators who need it can enable it.

Reply:
Yes, it's an option that can cause change and confusion, so I think we need to discuss more the default config, and there's also a way to make sure that enable is the default, but disabled on the testnet config

Reply:
Since most nodes are not validators we should go with default off. This is the same thing we do with `pex`, we turn it on by default, but it is recommended to have it off for validators.

Reply:
Now that it's off by default (thanks!), we can remove all of the `--consensus.double_sign_check_height 0` arguments in the test scripts and such.